content/footer/contents.lr

```diff
@@ -0,0 +1,11 @@
+_hidden: yes
+---
+enabled: yes
+---
+css_content_class: has-text-centered
+---
+content:
+
+**Slova Nova** by [Ethan Nielsen](https://slovanova.org).
+The source code is licensed [MIT](http://opensource.org/licenses/mit-license.php).
+The website content is licensed [CC BY NC SA 4.0](http://creativecommons.org/licenses/by-nc-sa/4.0/).
```

Comment:
The basic template here was taken from a random example in the Lektor docs.  I do NOT want to keep this content but I don't know what else to put put in the footer and the site looks a bit weird/unbalanced without that section.